content/specs/xcode-update-policy.md

```diff
@@ -0,0 +1,33 @@
+---
+description: How we release and deprecate Xcode versions
+title: Xcode update policy
+aliases:
+weight: 4
+---
+
+## Overview
+
+Codemagic macOS build machines come with a range of Xcode versions and runtimes preinstalled. 
+
+In general, we will always support the four latest major Xcode versions:
+
+* The latest major Xcode version and all of its minor versions at the latest patch version (e.g. Xcode `14.0.1`, `14.1`, `14.2` etc)
+* Three previous major Xcode versions at the latest `major.minor.patch` version (e.g. Xcode `13.4.1`, `12.5.1`, `11.7`)
+
+You can check the currently available Xcode versions [here](../specs/versions-macos).
+
+## Xcode version updates
+
+When a new beta or release version of Xcode is released, we aim to make it available to our users as soon as possible. We usually schedule the rollout of new images to a time with low demand to avoid build queues. 
```

Comment:
actually this is because network speed, but maybe better just say that schedule the rollout over weekend.